src/style.css

```diff
@@ -1,3 +1,73 @@
+:root {
+  --activeColour: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)